.gitignore

```diff
@@ -60,8 +60,15 @@ typings/
 # vscode settings
 .vscode
 
-# Custom redirects
-public/_redirects
+# macOS
+.DS_Store
 
 # Lighthouse reports
 *.report.html
+
+# Custom redirects
+public/_redirects
```

Comment:
This has to be added to the source, or a script should generate this on the fly.